lib/intercom-rails/encrypted_mode.rb

```diff
@@ -0,0 +1,34 @@
+module IntercomRails
+  class EncryptedMode
+    attr_reader :secret, :initialization_vector, :enabled
+
+    ENCRYPTED_MODE_SETTINGS_WHITELIST = [:app_id, :session_duration, :widget, :custom_launcher_selector, :hide_default_launcher, :alignment, :horizontal_padding, :vertical_padding]
+
+    def initialize(secret, initialization_vector, options)
```

Comment:
Could we just encapsulate the IV generation (through securerandom) and drop it from the list of the function args?